assets/js/components/event/ControlPanel.tsx

```diff
@@ -175,7 +175,7 @@ const useRerender = () => {
 
 type ControlPanelProps = {
   client: Client;
-  webrtc: MembraneWebRTC;
+  webrtc: MembraneWebRTC | null;
```

Comment:
I assume this is necessary to make a preview but this may allow the occurrence of the bug I've mentioned in the main review comment.

Reply:
We can add Tracks before we are connected to webrtc, so the potential bug is not really dependent on this.